### PR TITLE
Various updates and small fixes

### PR DESCRIPTION
**Pull request by SpadesNeil**
Map updates, gateway tweak, and a few fixes for some mobs that were behaving stupidly.

labyrinth.dmm is still not ready for use in the server so don't use it. I just inched its progress ahead slightly.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/animals/goose.dm [hunk 1]
```diff
@@ -55,4 +55,4 @@
 		custom_emote(1,"flaps and honks at [.]!")
 
 /mob/living/simple_animal/hostile/goose/Process_Spacemove(var/check_drift = 0)
-	return
+	return 1 // No drifting in space!
```
Comment (Arokha) [inline code/modules/mob/living/simple_animal/animals/goose.dm]: VOREStation Edit

Comment (SpadesNeil) [inline code/modules/mob/living/simple_animal/animals/goose.dm]: I thought the goose was just a Vorestation thing to begin with. :U

**Comment by SpadesNeil** [comment]
What got changed in Virgo-2? I did use the map merge tool but I fucked it up.

**Comment by SpadesNeil** [comment]
I just stole the Virgo-2.dmm file from https://github.com/VOREStation/VOREStation/pull/1369 because my changes to it are not important.

**Comment by Arokha** [comment]
The GitHub squash merge is generally horrible. It's disabled on our repo, but if you insist.

**Comment by Arokha** [comment]
In the future, do whatever, squash it yourself and let us know when you're ready for it to be merged (WIP label or something).

**Comment by SpadesNeil** [comment]
Why's it so bad on Github? D:

**Pull request merged**